src/styles/card-holder.scss

```
@@ -0,0 +1,27 @@
+.card-holder {
+  --column-count: 1;
+  padding: 50px 40px;
+
+  display: grid;
+  grid-template-columns: repeat(var(--column-count), $card-with);
+  gap: $cards-gap;
+  justify-content: center;
+
+  @media (min-width: 488px) {
+    & {
+      padding: 50px 20px;
+      --column-count: 2;
+    }
+  }
+  @media (min-width: 768px) {
```

```suggestion
  @media (min-width: 488px) {
    padding: 50px 20px;
    --column-count: 2;
  }
  
  @media (min-width: 768px) {
```
Actually, you don't need to repeat the selector.
Add blank lines between code blocks